README.rst

```diff
@@ -169,6 +169,12 @@ test
 
 `test` target executes tests.
 
+
+sync-to-source
+~~~~~~~~~~~~~~
+
+`sync-to-remote` target uploads the local files stored in `data` to specified data sources in such as S3 or NFS directories.
```

Comment:
> to specified data sources in such as S3 or NFS directories.

to specified data sources such as S3 or NFS directories.

Reply:
👍